Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -139,18 +125,31 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
   }
 
   // MARK: - Private Helpers
+
   private func configureHeaderFooterViews() {
-    let footerView = tableView.dequeueReusableHeaderFooterView(
-  withIdentifier: Nib.PaymentMethodsFooterView.rawValue
-      ) as? PaymentMethodsFooterView
-    footerView?.delegate = self
+    if let header = SettingsTableViewHeader.fromNib(nib: Nib.SettingsTableViewHeader) {
+      header.configure(with: Strings.Any_payment_methods_you_saved_to_Kickstarter())
+
+      let headerContainer = UIView(frame: .zero)
```

Comment:
I'm not sure but think this container (that doesn't use auto-layout) was missing and is the key...If only Apple could make this work in the next OS version 🤦‍♂️